content/en/users/im/cli/_index.md

```diff
@@ -0,0 +1,169 @@
+---
+title: "IM CLI"
```

Comment:
This is a bad title, never use acronyms in titles. See https://docs.egi.eu/about/contributing/style/
Moreover, use different text for the link in the left-side nav tree (e.g. by adding a line "linkTitle: Command Line") and the page title that can be more elaborate (e.g. "Infrastructure Manager Command Line Interface"). Then in the text the first time you write "command line interface " you immediately follow with the acronym "(CLI)".

Reply:
changed.